.travis.yml

```diff
@@ -2,6 +2,8 @@
 language: python
 python: "2.7.13"
 sudo: required
+dist: trusty
+group: deprecated-2017Q4
```

Comment:
I have included this because new travis image has problems with systemctl. I have opened this https://github.com/travis-ci/travis-ci/issues/8950